service/init.go

```diff
@@ -375,14 +286,70 @@ func (amf *AMF) Start() {
 		logger.InitLog.Errorf("initialise kafka stream failed, %v ", err.Error())
 	}
 
-	self := context.AMF_Self()
+	self := amfContext.AMF_Self()
 	util.InitAmfContext(self)
 	if self.EnableDbStore {
 		self.Drsm, err = util.InitDrsm()
 		if err != nil {
 			logger.InitLog.Errorf("initialise DRSM failed, %v", err.Error())
 		}
 	}
+	ctx, cancelServices := ctxt.WithCancel(ctxt.Background())
+
+	if factory.AmfConfig.Configuration.Telemetry != nil && factory.AmfConfig.Configuration.Telemetry.Enabled {
+		var tp *sdktrace.TracerProvider
+		tp, err = tracing.InitTracer(ctx, tracing.TelemetryConfig{
+			OTLPEndpoint:   factory.AmfConfig.Configuration.Telemetry.OtlpEndpoint,
+			ServiceName:    "amf",
+			ServiceVersion: factory.AmfConfig.Info.Version,
+			Ratio:          *factory.AmfConfig.Configuration.Telemetry.Ratio,
+		})
+		if err != nil {
+			logger.InitLog.Panic("could not initialize tracer", zap.Error(err))
+		}
+		logger.InitLog.Infoln("tracer initialized successfully")
+		defer func() {
+			err = tp.Shutdown(ctx)
+			if err != nil {
+				logger.InitLog.Error("failed to shutdown tracer", zap.Error(err))
```

Comment:
Why do you need to use `zap.Error`?

```suggestion
				logger.InitLog.Errorf("failed to shutdown tracer", err)
```